### PR TITLE
Added getBatteryVoltage() Method in OSTools

### PR DESCRIPTION
**Pull request by TheGreyDiamond**
Added getBatteryVoltage() Method in OSTools with human readybility argument. 
With getBatteryVoltage(human=True): 7.400 (float), without getBatteryVoltage(human=False): 7400 (int)

### CODE REVIEW

**Comment by XilefTech** [comment]
make your follow up function and then I'll see if we can anyhow use it... at this point pretty useless

**Comment by TheGreyDiamond** [comment]
Please recheck

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 __pycache__/
 *.pyc
 venv/
+
```
Comment (XilefTech) [inline .gitignore:4]: is that really necessary? you might wanna remove that from the pull request...

Comment (TheGreyDiamond) [reply .gitignore]: Done!

**Comment by XilefTech** [comment]
you might wanna use the log.warn() for the alert, since that got implemented recently

**Comment by XilefTech** [comment]
approved

**Pull request merged**